wp-content/themes/pub/wporg-learn-2020/front-page.php

```diff
@@ -35,7 +35,12 @@
 		<section>
 			<div class="row align-middle between section-heading">
 				<h2 class="h4 section-heading_title"><?php esc_html_e( 'Recent Courses', 'wporg-learn' ); ?></h2>
-				<a class="section-heading_link" href="/courses/"><span aria-hidden="true"><?php esc_html_e( 'View All Courses', 'wporg-learn' ); ?></span><span class="screen-reader-text"><?php esc_html_e( 'View All Courses', 'wporg-learn' ); ?></span></a>
+				<div class=section-heading">
```

Comment:
Thanks for adding these!

We're missing an opening `"` here, and this classname is intended to be a wrapper for the heading, rather than nested again.

I'll open a new PR to clean up a little.